Tests/SonarScanner.MSBuild.PreProcessor.Test/PreProcessorTests.cs

```diff
@@ -122,9 +122,36 @@ public void PreProc_EndToEnd_SuccessCase()
             mockServer.AssertMethodCalled("TryGetQualityProfile", 2); // C# and VBNet
             mockServer.AssertMethodCalled("GetRules", 2); // C# and VBNet
 
+            logger.DebugMessages.Should().Contain("Base branch parameter was not provided. Incremental PR analysis is disabled.");
+
             AssertAnalysisConfig(settings.AnalysisConfigFilePath, 2, logger);
         }
 
+        [TestMethod]
+        public async Task PreProc_WithPullRequestBranch()
+        {
+            const string baseBranch = "BASE_BRANCH";
+            var workingDir = TestUtils.CreateTestSpecificFolderWithSubPaths(TestContext);
+            var logger = new TestLogger();
+            var mockServer = new MockSonarQubeServer();
+            mockServer.Data.Languages.Add("cs");
```

Comment:
Is this line removed by the next PR? Removing it will allow to remove the empty line below too

Reply:
Yes, this disappears in the next PR.